pkg/comp-functions/functions/vshnnextcloud/deploy.go

```diff
@@ -640,22 +640,24 @@ func newValues(ctx context.Context, svc *runtime.ServiceRuntime, comp *vshnv1.VS
 		},
 	}
 
-	if image := svc.Config.Data["nextcloud_image"]; image != "" {
-		values["image"] = map[string]interface{}{
-			"repository": image,
-		}
-	}
-
+	// Chart 8.6.0+ uses separate registry and repository instead of combined image path
 	if registry := svc.Config.Data["imageRegistry"]; registry != "" {
-		image := fmt.Sprintf("%s/%s", registry, "xperimental/nextcloud-exporter")
+		// Nextcloud image
+		if repository := svc.Config.Data["nextcloud_image_repository"]; repository != "" {
```

Comment:
If for some reason we use the old configuration, this will simply be skipped and will result in using the default registry/repository configuration of the helm chart, possibly breaking instances.